src/store/dynamo.js

```diff
@@ -4,10 +4,6 @@ const dynamoose = require('dynamoose');
 const logger = require('../logging');
 const PromCollector = require('../metrics/PromCollector');
 
-if (!config.dynamo.tableName) {
```

Comment:
Same comment as for saveEntryAssureUnique, we need some sort of check when dynamo is not configured.

Reply:
nvm, just saw the added check to saveEntryAssureUnique.